board_client/animations/bouncing_pi.py

```
@@ -16,15 +16,62 @@
+def _draw_pi(graphics, x, y, pen):
+    graphics.set_pen(pen)
+    for row in range(PI_H):
+        for col in range(PI_W):
+            if PI_BITMAP[row][col]:
+                px = x + col
+                py = y + row
+                if 0 <= px < WIDTH and 0 <= py < HEIGHT:
+                    graphics.pixel(px, py)
+
+
+from collections import deque
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)